apps/oxlint/src/lib.rs

```diff
@@ -4,9 +4,12 @@ mod result;
 mod runner;
 mod walk;
 
-pub use crate::{
-    command::*,
-    lint::LintRunner,
-    result::{CliRunResult, LintResult},
-    runner::Runner,
-};
+pub mod cli {
```

Comment:
This module structure (nested `pub mod`) is inconsistent with my other module structures.

Reply:
This stack has entered rebase hell. I'll need to re-create it.